### ISSUE TITLE
logstash 2.1.0 error

### ISSUE THREAD

**Issue by jpcarey** [cyli/logstash-output-slack#12]
```
ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem "logstash-core":
  In snapshot (Gemfile.lock):
    logstash-core (= 2.1.0)

  In Gemfile:
    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-elasticsearch (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-elasticsearch (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-elasticsearch (>= 0) java depends on
      logstash-core (< 3.0.0, >= 2.0.0) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-http (>= 0) java depends on
      logstash-mixin-http_client (< 3.0.0, >= 2.0.2) java depends on
        logstash-core (< 3.0.0, >= 2.0.0.beta2) java

    logstash-output-slack (>= 0) java depends on
      logstash-core (< 2.0.0, >= 1.4.0) java

    logstash-core (= 2.1.0) java

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by JoshTGreenwood** [comment]
I'm getting this as well.  The latest version 0.1.3 has tighter gemspec restrictions and I believe they may need to be tweaked.

**Comment by JoshTGreenwood** [comment]
I think it's safe to use `core_upper_bound = "< 3.0.0"`

**Comment by augustin149** [comment]
I have the same issue, is there any walkaround?

**Comment by cyli** [comment]
I've merged @JoshTGreenwood's PR to bump the version (and also dropped JDK6 support, because it was getting tricky).  I've pushed a new release.  Does this fix the issue?

**Comment by JoshTGreenwood** [comment]
I'm still seeing a similiar error but I think I addressed it for good in #17, if you could take a look when you get time @cyli
